README.md

```diff
@@ -187,3 +187,225 @@ to MAIL FROM with
 and rejects incorrectly authenticated emails with [`reject_sender_login_mismatch`](reject_sender_login_mismatch) policy.
 `From:` header must correspond to envelope MAIL FROM,
 this is ensured by `filtermail` proxy.
+
+## Migrating chatmail server to a new host
+
+If you want to migrate your chatmail server to a new host,
+follow these steps:
+
+1. Block all ports except 80 and 22 with firewall on a new server.
+
+   To do this, add the following config to `/etc/nftables.conf`:
+```
+#!/usr/sbin/nft -f
+
+flush ruleset
+
+table inet filter {
+    chain input {
+        type filter hook input priority filter; policy drop;
+
+        # Accept ICMP.
+        # It is especially important to accept ICMPv6 ND messages,
+        # otherwise IPv6 connectivity breaks.
+        icmp type { echo-request } accept
+        icmpv6 type { echo-request, nd-neighbor-solicit, nd-router-advert, nd-neighbor-advert } accept
+
+        tcp dport { ssh, http } accept
+
+        ct state established accept
+    }
+    chain forward {
+        type filter hook forward priority filter;
+    }
+    chain output {
+        type filter hook output priority filter;
+    }
+}
+```
```

Comment:
how about we add a `cmdeploy run --disable-mx`  switch that makes sure the server is setup but not reachable/accepting SMTP/IMAP?  (not sure if simply not running postfix/dovecot might be enough?)

Reply:
I agree... not everyone knows how to adjust nftables. Admins will find an existing configuration at `/etc/nftables.conf`, modified by the ISP, their past self, or a team member. Adjusting nftables will much more easily lead to misconfiguration than simply stopping postfix + dovecot directly after initial setup in a `cmdeploy run --disable-mx` command.

If we leave out step 1 in favor of `cmdeploy run --disable-mx`, we could also swap (current) step 2 & 3 to keep the "registration doesn't work" timespan shorter.

Reply:
> If we leave out step 1 in favor of `cmdeploy run --disable-mx`, we could also swap (current) step 2 & 3 to keep the "registration doesn't work" timespan shorter.

no we can't, as step 2 is required for step 3's acmetool setup to work.

Reply:
Hm, if we simply copy `/var/lib/acme` in the beginning it should work.